common/windows/data/Makefile

```
@@ -0,0 +1,20 @@
+!include ..\delphi\Defines.mak
+
+BUILDUNIDATA_PATH=$(KEYMAN_ROOT)\common\windows\delphi\tools\buildunidata
+BUILDUNIDATA_PROGRAM=$(BUILDUNIDATA_PATH)\$(WIN32_TARGET_PATH)\buildunidata.exe
+
+DATA_SOURCE=$(KEYMAN_ROOT)\resources\standards-data\unicode-character-database
+DATABASE_PATH=$(KEYMAN_ROOT)\common\windows\bin\data
+DATABASE_FILENAME=$(DATABASE_PATH)\unicodedata.mdb
```

Doesn't really matter since it's running on Windows only, but to be consistent we could use correct casing...

```suggestion
DATABASE_FILENAME=$(DATABASE_PATH)\UnicodeData.mdb
```